packages/synapse-react-client/src/components/ChangePassword/ChangePassword.integration.test.tsx

```diff
@@ -551,4 +560,79 @@ describe('ChangePassword tests', () => {
     const otpInputs = await getTOTPInputs()
     expect(otpInputs).toHaveLength(6)
   })
+
+  it('supports requesting 2FA reset using a password', async () => {
```

Comment:
Critical test - when attempting to change your password, you can begin the 2FA reset process